vita3k/regmgr/src/regmgr.cpp

```diff
@@ -346,26 +372,19 @@ int32_t get_int_value(RegMgrState &regmgr, const std::string &category, const st
     std::lock_guard<std::mutex> lock(regmgr.mutex);
 
     const auto cat = fix_category(category);
-    const auto &sys = regmgr.system_dreg[cat][name];
-    auto value_str = std::string(sys.begin(), sys.end());
-    if (!std::all_of(value_str.begin(), value_str.end(), [](unsigned char c) { return std::isdigit(c) || (c == '\0'); })) {
-        LOG_ERROR("Invalid value for {}{}: {}, attempt using default value!", cat, name, value_str);
-        value_str = set_default_value(regmgr, cat, name);
-        if (value_str.empty())
-            return 0;
-    }
 
-    return string_utils::stoi_def(value_str);
+    // Convert hex string to int
+    return byte_swap(*reinterpret_cast<const uint32_t *>(regmgr.system_dreg[cat][name].data()));
```

Comment:
Byte swap do not do anything with hex. It switch endianness. Delete your comment, it's misleading. (same further)

Reply:
OK if you said